src/xrEngine/defines.h

```diff
@@ -47,7 +47,8 @@ enum
 
     rsShowFPS = (1ul << 24ul), // 24 was reserved for editor
     rsDrawParticles = (1ul << 25ul), // 25 was reserved for editor
-    // 26-32 bit - reserved to Editor
+    rsDisableManagedTex = (1ul << 26ul), // 26 was reserved for editor
```

Comment:
Agree.

Reply:
Well... it was.

I need to store rsDisableManagedTex in FlagsX since it's required for console command to work properly.

There only CCC_Mask in Console (wich works with flags), and no CCC_Bool (maybe i missed something). 

It was CCC_Integer (0\1) previosly, but vanila UI-checkboxes in game options can't handle such things.
_______

So if i can't put it to psDeviceFlags. the i need to create entirely new Flags value only to store one option. Or cut off console command completely.

Reply:
Yeah, you can solve this by making an additional renderer command (see the comment from @Xottab-DUTY below)